S23/mepripri/ToLowerCase.py

```diff
@@ -0,0 +1,49 @@
+import csv
+
+def splitLang(l):
+    l=l.split('-')
+    if len(l[0])==2:
+        l=l[0]
+    return l
+
+def toLower(s,lang):
+    str=''
+    first=0
+    lang=splitLang(lang)
+    if lang=='tr' or lang=='az':
+        for i in s:
+            if ord(i)==73:
```

Comment:
While this may work, I think the use of hard coded numbers is not ideal. Maybe if the function merely searches for 'I' instead of using ord() that would be a little clearer.

Reply:
Yes, that also perfect that will be a direct search.